site/docs/v1/tech/client-libraries/index.adoc

```diff
@@ -1,39 +1,70 @@
 ---
 layout: doc
-title: Client Libraries Overview
-description: An overview of the FusionAuth Client Libraries
+title: Client Libraries and SDKs Overview
+description: An overview of the FusionAuth Client Libraries and SDKs
 navcategory: developer
 ---
 
 :sectnumlevels: 0
 
 == Overview
 
-Client libraries will help you quickly integrate your application with FusionAuth. All of our client libraries are open source and hosted on our https://github.com/FusionAuth[GitHub] account. You can fork and tweak them as well as look over the code to learn how the client libraries work.
+Client libraries and SDks will help you quickly integrate your application with FusionAuth. All of our client libraries are open source and hosted on our https://github.com/FusionAuth[GitHub account]. You can fork and tweak them as well as look over the code to learn how the client libraries work.
 
+* <<SDKs vs Client Libraries>>
 * <<Languages>>
 * <<Usage Suggestions>>
 * <<Versioning>>
 
+== SDKs vs Client Libraries
+
+A **client library** is a thin wrapper over our FusionAuth APIs. It needs to cover all FusionAuth APIs. A client library is like a set of legos, to be put together by a developer who wants to extend or manipulate FusionAuth to meet their needs.
```

Comment:
I might just say "At FusionAuth a client library..." just in case some opinionated people disagree with our usage